pelican/utils.py

```diff
@@ -646,22 +638,37 @@ def process_translations(content_list, order_by=None):
                 for x in lang_items:
                     logger.warning('\t%s', x.source_path)
 
+        # items with `translation` metadata will be used as translations...
+        candidate_items = list(filter(
+            lambda i:
+                i.metadata.get('translation', 'false').lower() == 'false',
+            items))
+        # ...unless all items with that slug are translations
+        if not candidate_items:
+            logger.warning('All times of "%s" are translations', slug)
+            candidate_items = items
+
         # find items with default language
-        default_lang_items = list(filter(
+        original_items = list(filter(
             attrgetter('in_default_lang'),
-            default_lang_items))
+            candidate_items))
+
+        # if there is no article with default language, go back one step
+        if not original_items:
+            original_items = candidate_items
 
-        # if there is no article with default language, take an other one
-        if not default_lang_items:
-            default_lang_items = items[:1]
+        # display warning if there are several base items
+        if len(original_items) > 1:
+            logger.warning('There are %s original (not translated) items "%s"',
+                           len(original_items), slug)
 
         if not slug:
             logger.warning(
```

Comment:
Also, in that case, the other log message [line 647](https://github.com/getpelican/pelican/pull/2052/files#diff-fb2a57e0cb7b201d7a478128b53c1c5eL647) should have already shown all sources that have that slug, so I think we are good.